tools/hidl/hidlcomp.cpp

```diff
@@ -4060,6 +4062,8 @@ void EspServInfo::write_esp_binding_ipp()
         outf("#endif\n");
     }
 
+    outs("\tstd::map<std::string, std::string> m_qualifiedMethodNames;\n");
```

Comment:
This would be better off in CHttpSoapBinding, along with the function qualifyMethodName().

It would avoid duplication, and make the code easier to review - since it would be compiled.